input/templates/partials/catalog/reserve/initialize-map-scripts.hbs

```diff
@@ -0,0 +1,79 @@
+<script>
+  $(window).load(function() {
```

Comment:
Why is this files a template and not js? In general scripts should be loaded as resource so that browser cam cache them.